### DISCUSSION TITLE
wemos / philips hue emulation for controlling Amazon Echo

### DISCUSSION THREAD

**Discussion by dodacs** [HASwitchPlate/openHASP#349] [Ideas]
Tasmota has has the option to emulate wemos (one relays) or philips hue (more than one relays) on the firmware. It would be useful to have this code on openHASP and control the plate with Amazon Echo devices. I know it can be done through Home Assistant, but I know people that would use the plates without any home automation server.

**Comment by fvanroie** [comment]
Thanks for this idea. There are so many Home Automation protocols and companies out there, that supporting them all would be very time consuming. The concept for openHASP is that the intelligence of the plate is offloaded *as much as possible* to a specialized Home Automation controller.

It can probably be added as a [Custom Interface](https://www.openhasp.com/0.6.3/compiling/custom/), but I haven't got any Cloud connected devices that I could test with. So it would be up to a user/developer to add it in.
